docs/gke/enable_iap.sh

```diff
@@ -68,6 +68,15 @@ gcloud --project=${PROJECT} compute backend-services update ${BACKEND_SERVICE} \
       --global \
       --iap=enabled,oauth2-client-id=${CLIENT_ID},oauth2-client-secret=${CLIENT_SECRET}
 
+while [[ -z ${HEALTH_CHECK_URI} ]];
+  do HEALTH_CHECK_URI=$(gcloud compute --project=${PROJECT} health-checks list --filter=name~k8s-be-${NODE_PORT}- --uri);
+  echo "Waiting for the healthcheck resource PROJECT=${PROJECT} NODEPORT=${NODE_PORT} SERVICE=${SERVICE}...";
+  sleep 2;
+done
+
+# Update the healthcheck request path to "/healthz" since GCP load balancer defaults to "/"
```

Comment:
Please expand the comment to explain that this is to deal with the case in which ingress is created before the backing deployment the health check won't be set correctly.